native/consumer.cc

```diff
@@ -1142,8 +1177,9 @@ Napi::Value PactffiWithMultipartFile(const Napi::CallbackInfo& info) {
   std::string contentType = info[2].As<Napi::String>().Utf8Value();
   std::string file = info[3].As<Napi::String>().Utf8Value();
   std::string partName = info[4].As<Napi::String>().Utf8Value();
+  std::string boundary = extractBoundary(contentType);
 
-  StringResult res = pactffi_with_multipart_file(interaction, part, contentType.c_str(), file.c_str(), partName.c_str());
+  StringResult res = pactffi_with_multipart_file_v2(interaction, part, contentType.c_str(), file.c_str(), partName.c_str(), boundary.empty() ? nullptr : boundary.c_str());
```

Comment:
passing boundary as nullptr if unable to extract as the reference pactffi_with_multipart_file function under pact-reference was using a nullptr for missing argument.

<img width="1146" height="747" alt="Image" src="https://github.com/user-attachments/assets/50015903-bbb2-4bcd-b295-26210539d92d" />